src/services/currencyService.js

```diff
@@ -0,0 +1,26 @@
+import { Store } from '../store'
+import { RequestBuilder } from '../utils/requestBuilder'
+
+export class CurrencyService {
+  #store
+
+  #http
+
+  #url = 'https://api.coinstats.app/public/v1/coins?skip=0&limit=5&currency=EUR'
```

Comment:
Create directory environments in src, create index.js in this folder and add object with same name, add key API_URL and variable this string.
For example, 
`
const environments = {
  API_URL: 'https://api.coinstats.app/public/v1',
}
`
and use it in class
`
export class CurrencyService {
  ...
  #url = '${environments.API_URL}/coins?skip=0&limit=5&currency=EUR'
}
`